### PR TITLE
Added the flags for optimizing the kotlin

### PR DESCRIPTION
**Pull request by pokk**
as the title.

### CODE REVIEW

File: mobile-ui/build.gradle [hunk 1]
```diff
@@ -86,6 +86,8 @@ android {
 
 kapt {
     correctErrorTypes = true
+    useBuildCache = true  // Gradle version 4.3 and above is required
```
Comment (lekaha) [inline mobile-ui/build.gradle:89]: You are using 4.3??

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx1536m
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+org.gradle.daemon=true
+kotlin.incremental=true
+kotlin.incremental.usePreciseJavaTracking=true
```
Comment (lekaha) [inline gradle.properties:16]: Can you give some explanation of these configs?

Comment (pokk) [reply gradle.properties:16]: They are from my previous project with `Java` code, I forgot removing it.

File: mobile-ui/build.gradle (hunk 1 above)
Comment (pokk) [reply mobile-ui/build.gradle:89]: I'm not in 4.3, it doesn't work now. I just put it in temporally. Once our gradle version is upgraded to 4.3 we can use it directly.

**Pull request merged**